### ISSUE TITLE
ally addon not working

### ISSUE THREAD

**Issue by ajitbohra** [storybooks/storybook#5078]
**Describe the bug**
Ally addon no checks are run
**To Reproduce**
- Add decorator globally

**Expected behavior**
Addon tab show ally checks passed and failed

**Screenshots**
![story](https://user-images.githubusercontent.com/1039236/50398395-3f9ca080-079d-11e9-99ef-ca616c0e98a0.png)

**System:**
 - OS: ubuntu 18.10
 - Device: Desktop
 - Browser: Chromium
 - Framework: react
 - Addons: ally
 - Version: storybook@4.1.3 / ally-addon@4.1.3

**Live URL**
https://wp-storybook.netlify.com

**Comment by ogonkov** [comment]
Same here

**Comment by andyWynyard** [comment]
I also have this issue. 

I've had a look at older projects with older versions `4.0.0-rc.1` with no issue.

System:

- OS: Mac OS
- Device: Desktop
- Browser: Chromium
- Framework: next/react
- Addons: ally
- Version: storybook@4.0.7 / ally-addon@4.0.7

**Comment by igor-borisov** [comment]
I think it is the same issue #4889

**Comment by shilman** [comment]
This should be fixed in https://github.com/storybooks/storybook/releases/tag/v4.2.0-alpha.10

Please give it a try and comment here or re-open this issue if it's still not working for you. Thanks for your help!
